### ISSUE TITLE
use FlashLFQ for estimates of PTM occupancy

### ISSUE THREAD

**Issue by trishorts** [smith-chem-wisc/MetaMorpheus#1565]
rather than PSMs.

**Comment by lonelu** [comment]
A peptide may have a very different intensity than the peptide with modification. Depend on what the mod is.

**Comment by lonelu** [comment]
I agree this will be easier to code and the result may be more accurate than counting PSMs.

**Comment by acesnik** [comment]
I'm very interested in this!

I wonder if we could estimate the difference in ionization efficiency based on comparing all unmodified <-> singly modified peptide intensity ratios.
